### PR TITLE
docs: revised openapi-cli overview page

### PR DESCRIPTION
**Pull request by bandantonio**
## What/Why/How?

Updated the home page for the `openapi-cli` tool.

This PR contains only the basic information that the first-time reader needs to get familiar with the tool.
Deleted parts will be added in the upcoming PRs.

## Check yourself

- [ ] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
+- ✅ Multi-file validation. No need to bundle your files before validation.
+- ✅ Lightning-fast validation. Check a 1 MB file in less than one second.
+- ✅ Support for remote `$ref`s. Reference a definition hosted on any location.
+- ✅ Bundle a multi-file definition into a single file for compatibility with external tools.
+- ✅ Configurable severity levels for each rule. Fine tune Redocly OpenAPI CLI to suit your needs.
+- ✅ Human-readable error messages. Now with stacktraces and codeframes.
+- ✅ Intuitive suggestions for misspelled types or references. No error will pass you by.
+- ✅ Easy-to-implement custom rules. Need something? Ask us or do it yourself.
+- ✅ Preview reference docs. Writing OpenAPI documents has never been easier.
+- ✅ Support for OAS 3.0 and Swagger 2.0 (OAS 3.1 is coming soon).
```
Comment (adamaltman) [inline docs/index.md:35]:
```suggestion
- ✅ Support for OAS 3.0 and Swagger 2.0 (OAS 3.1 is coming soon).
```
3.1 is now here

**Review by skadinna — DISMISSED** [review]
This looks pretty good! :+1:

I'm going to approve it, but I wouldn't merge it until we have a new page with installation instructions ready in another PR. If we merge it now, we won't have any installation instructions at all in the docs. Does that make sense to you @swapnilogale?

File: docs/index.md (hunk 1 above)
Comment (bandantonio) [reply docs/index.md]: Thanks @adamaltman . Fixed

**Comment by swapnilogale** [comment]
@bandantonio once you resolve the merge conflict (like we discussed last night), this PR is good to go. https://github.com/Redocly/openapi-cli/pull/337 and https://github.com/Redocly/openapi-cli/pull/338 have been merged!

**Review by skadinna — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**